OsmAnd/src/net/osmand/aidl/map/SetMapLocationParams.java

```diff
@@ -8,12 +8,18 @@
 	private double latitude;
 	private double longitude;
 	private int zoom;
+	private float rotation;
 	private boolean animated;
 
 	public SetMapLocationParams(double latitude, double longitude, int zoom, boolean animated) {
+		this(latitude, longitude, zoom, Float.NaN, animated);
```

Comment:
The second constructor is not necessary for backwards compatibility, the only order of written and read values is important, except for some situations with lists of items.